src/basic/selinux-util.c

```diff
@@ -57,6 +57,11 @@ bool mac_selinux_use(void) {
 
 bool mac_selinux_enforcing(void) {
 #if HAVE_SELINUX
+        PROTECT_ERRNO;
+
+        /* Check for policy change so enforcing state is kept up-to-date by callback */
+        (void) avc_netlink_check_nb();
```

Comment:
so, doing this check will do a few syscalls I presume? But is that really quicker than just calling security_getenforce() right away which pressumably also just does a few syscalls?

why is this function call placed here? what does it have to do with a check for "are we in enforcing mode"?